### DISCUSSION TITLE
Does Ory support SaaS companies that need multiple third party auth providers (e.g. Okta/Auth0)?

### DISCUSSION THREAD

**Discussion by djMax** [ory/kratos#3210] [Q&A]
I believe things like Cognito support this, but would it be possible to use Ory to manage identity for a company that wanted to enable their customers (employees of other companies) to use THEIR OWN IDP? i.e my company is A, I'm selling to B and C. B uses Okta, C uses Auth0, I want an application where unaffiliated people can auth with passwordless, but B folks would use Okta and C folks Auth0. Make sense?

And is it possible in Ory?

**Comment by kmherrmann** [comment]
This is easiest on Ory Network, where you can create a project per customer (B, C) and configure individual IDP connections in each.
